common/traits/wc_race_traits.txt

```diff
@@ -1978,6 +1978,23 @@ creature_ancient_wind = {
 	random = no
 	customizer = no
 }
+creature_ancient_coral = {
+	diplomacy = -3
+	learning = 3
+	stewardship = 2
+	health = 1
+	combat_rating = 60
+
+	same_opinion = 10
+	general_opinion = -10
+	druidism_group_opinion = 5	# Children of Nature
+
+	text_effect_average_lifespan_immortality = 1
+	immortal = yes
+
+	random = no
+	customizer = no
+}
```

Comment:
They're fertile. Is it intented?

Reply:
corals are animals and reproduce sexually